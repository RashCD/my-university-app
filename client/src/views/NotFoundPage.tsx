import { RouteComponentProps } from '@reach/router';
import Styles from '../assets/styles/views/NotFoundPage.module.scss';
import AppHeader from '../components/AppHeader';

const NotFoundPage = (props: RouteComponentProps) => {
  return (
    <div className="app">
      <AppHeader />
      <div className={Styles.notFound}>404 | Not found</div>
    </div>
  );
};

export default NotFoundPage;
