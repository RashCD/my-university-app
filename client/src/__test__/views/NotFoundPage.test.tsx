import { render, cleanup } from '@testing-library/react';
import NotFoundPage from '../../views/NotFoundPage';

afterEach(cleanup);

describe('Test for not found page page', () => {
  test('should match snapshot of not found page page', () => {
    const container = render(<NotFoundPage />);

    expect(container).toMatchSnapshot();
  });
});
