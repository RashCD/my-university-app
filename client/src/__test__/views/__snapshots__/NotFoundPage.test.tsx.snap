// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for not found page page should match snapshot of not found page page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="app"
      >
        <header
          class="appHeader"
        >
          <h1
            class="title"
          />
          <button
            class="button loginButton"
            type="button"
          >
            <span
              class="iconContainer"
              data-icon="true"
              style="--icon-src: url(user.svg); --icon-color: var(--white); --icon-size: 30px;"
            />
            Login
          </button>
        </header>
        <div
          class="notFound"
        >
          404 | Not found
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="app"
    >
      <header
        class="appHeader"
      >
        <h1
          class="title"
        />
        <button
          class="button loginButton"
          type="button"
        >
          <span
            class="iconContainer"
            data-icon="true"
            style="--icon-src: url(user.svg); --icon-color: var(--white); --icon-size: 30px;"
          />
          Login
        </button>
      </header>
      <div
        class="notFound"
      >
        404 | Not found
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
